modules/@apostrophecms/piece-type/index.js

```diff
@@ -205,6 +205,9 @@ module.exports = {
       return doc;
     },
     async post(req) {
+      if (Object.keys(req.body).length === 0) {
+        return self.newInstance();
+      }
```

Comment:
I realize this won't save anything to the db, but we should probably put it after `self.publicApiCheck(req);` anyway.

Reply:
Bea is correct. Also, we should check for the presence of a parameter like _newInstance rather than assuming there can never be a piece type that can legitimately be created without parameters.